clientGQL_test.go

```diff
@@ -166,7 +166,11 @@ func RegisterPaginatedEndpoint(t *testing.T, endpoint string, requests ...TestRe
 	url := fmt.Sprintf("/LOCAL_TESTING/%s", endpoint)
 	requestCount := 0
 	autopilot.Mux.HandleFunc(url, func(w http.ResponseWriter, r *http.Request) {
-		GraphQLQueryTemplatedValidation(t, requests[requestCount].RequestWithVariables())(r)
+		GraphQLQueryTemplatedValidation(t, fmt.Sprintf(
```

Comment:
likewise - we need to adjust autopilot first so that we can pass in just `requests[requestCount]` and not have to do the `fmt.Sprintf(` stuff.

choosing to use the more structured way is the better thing long term.